catalog/installGit.sh

```diff
@@ -19,9 +19,9 @@ waitForAll
 install "$CATALOG_HOME/github/webhook.js" \
     github/webhook \
     -a feed true \
-    -a description 'Creates a webhook on github to be notified on selected changes' \
-    -a parameters '[ {"name":"username", "required":true, "bindTime":true}, {"name":"repository", "required":true, "bindTime":true}, {"name":"accessToken", "required":true, "bindTime":true},{"name":"events", "required":true} ]' \
-    -a sampleInput '{"username":"whisk", "repository":"WhiskRepository", "accessToken":"123ABCXYZ", "events": "push,commit,delete"}'
+    -a description 'List, Create, Delete webhook on github to be notified on selected changes' \
+    -a parameters '[ {"name":"username", "required":true, "bindTime":true}, {"name":"repository", "required":true, "bindTime":true}, {"name":"accessToken", "required":true, "bindTime":true},{"name":"events", "required":true}, {"name":"triggerName", "required":false, "bindTime":true}, {"name":"lifecycleEvent", "required":true, "bindTime":true}, {"name":"hookid", "required":false, "bindTime":true}, {"name":"endpoint","required":true, "bindTime":true}]' \
+    -a sampleInput '{"username":"whisk", "repository":"WhiskRepository", "accessToken":"123ABCXYZ", "events": "push,commit,delete", "lifecycleEvent":"create", "triggerName":"whisk.system/github","endpoint":"openwhisk.ng.bluemix.net"}'
```

Comment:
You are using the triggername as additional parameter. Why don't you use the one, that is already provided by default?